internal/services/netapp/netapp_volume_resource_test.go

```diff
@@ -390,6 +406,34 @@ resource "azurerm_netapp_volume" "test_snapshot_vol" {
 `, template, data.RandomInteger)
 }
 
+func (NetAppVolumeResource) nfsv3SnapshotDirectoryVisibleFalse(data acceptance.TestData) string {
+	template := NetAppVolumeResource{}.template(data)
+	return fmt.Sprintf(`
+%[1]s
```

Comment:
This needs to have a blank line after it to don't break TF linting, the end result is that %[1]s will be replaced with part of the HCL template and this will give no spacing between resources.